src/renderer/pkijs/index.ts

```diff
@@ -0,0 +1,48 @@
+import { signing } from './tests/sign'
+import { extractPubKey } from './tests/extractPubKey'
+import { verificationSignature } from './tests/verification'
+
+import { createRootCA } from './generatePems/generateRootCA'
+import { createUserCsr } from './generatePems/requestCertificate'
+import { createUserCert } from './generatePems/generateUserCertificate'
+import { verifyUserCert } from './generatePems/verifyUserCertificate'
+
+const createPem = async ({ rootCert, userCert }) => {
```

Comment:
lets try to clean up this pull request from helper scripts you used for debugging